platform/features/timeline/src/capabilities/GraphCapability.js

```diff
@@ -55,12 +55,14 @@ define(
                 if (domainObject.getModel().type === 'timeline' &&
                         result.power &&
                             domainObject.getModel().capacity > 0) {
+                    domainObject.getModel().startingSOC =
+                        parseFloat(domainObject.getModel().startingSOC) || 100;
```

Comment:
Direct modification of domain object models is [unsupported](http://nasa.github.io/openmct/docs/guide/#modifying-models) and may have unexpected consequences. I'd assign this to a `var` instead.

Also, I think there's a logic bug here; if the user has entered 0 for SOC, that will be falsy here, so this would fall back to 100 instead of whatever the user entered.